### DISCUSSION TITLE
xmlns:util not found

### DISCUSSION THREAD

**Discussion by alicera** [wixtoolset/issues#6984] [Questions]
https://www.hass.de/content/wix-how-use-removefolderex-your-xml-scripts

xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
I set the function and build it .
It show the xmlns:util not found

error CNDL0200 : The Component element contains an unhandled extension element 'util:RemoveFolderEx'. Please ensure that the extension for elements in the 'http://schemas.microsoft.com/wix/UtilExtension' namespace has been provided.

**Comment by alicera** [comment]
xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">

**Comment by wyrdfish** [comment]
The article you posted has the answers, are you following all the steps?
You need to tell the compiler to use the extension, with with a reference in visual studio or on the command  line.

https://wixtoolset.org/documentation/manual/v3/howtos/general/extension_usage_introduction.html

**Comment by alicera** [comment]
if I use it , how to include the  extension
 set(CPACK_WIX_TEMPLATE ${CMAKE_CURRENT_SOURCE_DIR}/cmake/wix.template.in)



https://stackoverflow.com/questions/46362937/wix-utilremovefolderex-does-not-work

**Reply by robmen** [reply]
You should go ask CPACK.
